src/main/java/net/atos/client/zgw/zrc/model/RelevanteZaak.java

```diff
@@ -42,4 +42,8 @@ public URI getUrl() {
     public AardRelatie getAardRelatie() {
         return aardRelatie;
     }
+
+    public boolean is(final URI url, final AardRelatie aardRelatie) {
```

Comment:
Waarom maak je hier geen equals methode van?

Reply:
Omdat ik in een geval wel een ander object heb maar in het andere geval niet. Dan zou ik daar een object moeten aanmaken wat ik verder niet nodig heb behalve voor de vergelijking.